### ISSUE TITLE
Depends nodejs (>=5.7)

### ISSUE THREAD

**Issue by Xykon** [UDOOboard/udoo-web-conf#15]
Another thing I have noticed when building the packages is the dependency on nodejs (>=5.7). It seems nodejs is now using different versions so I've removed the version dependency in debian/control file completely.

Current version of nodejs package on my udoo-neo is 0.12.12-1nodesource1~trusty1

**Comment by ektor5** [comment]
I tried 0.12 and 4.x  on this version of udoo-web-conf and it didn't start. Please update the deb source in `/etc/apt/sources.list.d/nodejs.list` to 5.x

**Comment by Xykon** [comment]
It worked fine for me... UDOObuntu 2 RC1 fresh download except using my own repository http://www.tiny-dev.com/udoo/ to install webconf packages. This is from my Packages file (installs and runs without problems):

Package: udoo-web-conf
Version: 0.3.1
Architecture: all
Maintainer: Ettore Chimenti ek5.chimenti@gmail.com
Installed-Size: 7368
Depends: sysv-rc (>= 2.88dsf-24) | file-rc (>= 0.8.16), udoo-web-docs, nodejs (>= 0.12), udoo-web-conf-modules (= 0.3.1), network-manager, dtweb
Homepage: http://www.udoo.org
Priority: optional
Section: utils
Filename: pool/main/u/udoo-web-conf/udoo-web-conf_0.3.1_all.deb
Size: 2285958
SHA256: d5801e80577597b6ddde8876c1993cc3d9ac93b3da4b368f546475c86b7bca60
SHA1: 6c42f8ecc8134a5a195db99bcb355ffa4fc314c5
MD5sum: dea78b0f9bf6ac0c0113af067142d930
Description: UDOO Neo Web Configurator
 This tool is designed to allow easy configuration and act as an handy web-based
 control panel for UDOO NEO.

Will try to update /etc/apt/sources.list.d/nodejs.list and see if that causes any problems.

Update: I did upgrade to nodejs 5.0 using 
`curl -sL https://deb.nodesource.com/setup_4.x | sudo -E bash -`
`sudo apt-get install -y nodejs`
but now the webinterface isn't loading (probably because I built it using the older 0.12 version).

Is any other official part  of udoo-neo using nodejs? Should I hold off upgrading nodejs in my production image until the upgrade is officially pushed through?
